sentry-samples/sentry-samples-spring-boot-jakarta-opentelemetry/src/main/java/io/sentry/samples/spring/boot/jakarta/SentryDemoApplication.java

```diff
@@ -42,37 +31,39 @@ RestClient restClient(RestClient.Builder builder) {
     return builder.build();
   }
 
-  @Bean
-  public JobDetailFactoryBean jobDetail() {
-    JobDetailFactoryBean jobDetailFactory = new JobDetailFactoryBean();
-    jobDetailFactory.setJobClass(SampleJob.class);
-    jobDetailFactory.setDurability(true);
-    jobDetailFactory.setJobDataAsMap(
-        Collections.singletonMap(SENTRY_SLUG_KEY, "monitor_slug_job_detail"));
-    return jobDetailFactory;
-  }
-
-  @Bean
-  public SimpleTriggerFactoryBean trigger(JobDetail job) {
-    SimpleTriggerFactoryBean trigger = new SimpleTriggerFactoryBean();
-    trigger.setJobDetail(job);
-    trigger.setRepeatInterval(2 * 60 * 1000); // every two minutes
-    trigger.setRepeatCount(SimpleTrigger.REPEAT_INDEFINITELY);
-    trigger.setJobDataAsMap(
-        Collections.singletonMap(SENTRY_SLUG_KEY, "monitor_slug_simple_trigger"));
-    return trigger;
-  }
+  //  @Bean
+  //  public JobDetailFactoryBean jobDetail() {
+  //    JobDetailFactoryBean jobDetailFactory = new JobDetailFactoryBean();
+  //    jobDetailFactory.setJobClass(SampleJob.class);
+  //    jobDetailFactory.setDurability(true);
+  //    jobDetailFactory.setJobDataAsMap(
+  //        Collections.singletonMap(SENTRY_SLUG_KEY, "monitor_slug_job_detail"));
+  //    return jobDetailFactory;
+  //  }
+  //
+  //  @Bean
+  //  public SimpleTriggerFactoryBean trigger(JobDetail job) {
+  //    SimpleTriggerFactoryBean trigger = new SimpleTriggerFactoryBean();
+  //    trigger.setJobDetail(job);
+  //    trigger.setRepeatInterval(2 * 60 * 1000); // every two minutes
+  //    trigger.setRepeatCount(SimpleTrigger.REPEAT_INDEFINITELY);
+  //    trigger.setJobDataAsMap(
+  //        Collections.singletonMap(SENTRY_SLUG_KEY, "monitor_slug_simple_trigger"));
+  //    return trigger;
+  //  }
+  //
+  //  @Bean
+  //  public CronTriggerFactoryBean cronTrigger(JobDetail job) {
+  //    CronTriggerFactoryBean trigger = new CronTriggerFactoryBean();
+  //    trigger.setJobDetail(job);
+  //    trigger.setCronExpression("0 0/5 * ? * *"); // every five minutes
+  //    return trigger;
+  //  }
 
-  @Bean
-  public CronTriggerFactoryBean cronTrigger(JobDetail job) {
-    CronTriggerFactoryBean trigger = new CronTriggerFactoryBean();
-    trigger.setJobDetail(job);
-    trigger.setCronExpression("0 0/5 * ? * *"); // every five minutes
-    return trigger;
-  }
-
-  @Bean
-  public Tracer getTracer() {
-    return GlobalOpenTelemetry.getTracer("tracerForSpringBootDemo");
-  }
+  //  @Bean
+  //  public Tracer getTracer(OpenTelemetry openTelemetry) {
+  //    GlobalOpenTelemetry.set(openTelemetry);
+  //    return openTelemetry.getTracer("tracerForSpringBootDemo");
+  ////    return GlobalOpenTelemetry.getTracer("tracerForSpringBootDemo");
+  //  }
```

Comment:
Should this be re-enabled or removed?

Reply:
No, there's different code we want. updated in a follow up PR